src/main/java/com/vint/shop/service/RoleService.java

```diff
@@ -0,0 +1,8 @@
+package com.vint.shop.service;
+import com.vint.shop.domain.Role;
+
+public interface RoleService {
+    Role findRoleByName(String role);
```

Comment:
No need to include "Role" in method name as it's pretty clear. Let's name it `findByName`

Reply:
delete thise service and all metods in repository